### ISSUE TITLE
luci-app-mwan3: Looking for `ip` in the wrong path

### ISSUE THREAD

**Issue by luke-jr** [openwrt/luci#1385]
With luci-app-mwan3 installed, my syslog regularly gets:

```
Thu Oct  5 03:56:04 2017 daemon.err uhttpd[2271]: sh: /usr/bin/ip: not found
Thu Oct  5 03:57:46 2017 daemon.err uhttpd[2271]: uci: Entry not found
```

**Comment by spacezorro** [comment]
FYI.. This was fixed in 0999f64

**Comment by feckert** [comment]
@hnyman i think we could close this issue
